tests/forms/FormFieldTest.php

```diff
@@ -159,6 +159,9 @@ public function testReadonly() {
 		$this->assertContains('readonly="readonly"', $field->getAttributesHTML());
 		$field->setReadonly(false);
 		$this->assertNotContains('readonly="readonly"', $field->getAttributesHTML());
+		$this->assertInstanceOf('FormField', $field->performReadonlyTransformation());
+		eval("class FormField_Readonly extends FormField {}");
```

Comment:
@ntd Could you create two formfield subclasses at the bottom of that test file and use those? `FormFieldTest_TestField` and `FormFieldTest_TestField_Readonly`, for example?